starter-code/iteration2/blog/src/app/main/main.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-main',
+  templateUrl: './main.component.html',
+  styleUrls: ['./main.component.css']
+})
+export class MainComponent implements OnInit {
+
+    title = 'Learn how to code with Ironhack';
```

Comment:
Aquí es más correcto hacerlo tal que title: String;

Y luego en el constructor, le añades el valor this.title = 'Learn how to code with Ironhack';

Así se "respetaría" el añadir la interfaz previa instacia de la clase 😄